src/partner/modules/EventList/EventListItem.tsx

```diff
@@ -147,20 +119,17 @@ export class EventListItem extends React.Component<IEventItemProps, IEventItemSt
             </tr>
           </tfoot>
         </ListStyled.Table>
-        <Modal title="Edit Event" show={this.state.editEvent} closeModal={this.showModalEditEvent}>
-          <CreateEvent editEvent={true} eventInfo={this.props.eventInfo} />
-        </Modal>
         <Modal
-          title="Cancel Event"
-          show={this.state.cancelEvent}
-          closeModal={this.closeModalCancelEvent}
+          title="Edit Event"
+          show={this.state.editEvent}
+          closeModal={() => this.setState({ editEvent: false })}
         >
-          <ListStyled.H2>Are you sure you want to cancel this event?</ListStyled.H2>
-          <ListStyled.RowData>
-            <ListStyled.GradientButton onClick={this.handleCancelEvent}>
-              Confirm
-            </ListStyled.GradientButton>
-          </ListStyled.RowData>
+          <CreateEvent
+            editEvent={true}
+            eventInfo={this.props.eventInfo}
+            onEdit={this.props.onEdit}
+            closeModal={() => this.setState({ editEvent: false })}
```

Comment:
if you are using `class` based components always favor `methods` over `inline-functions`